Mathlib/Data/Matrix/Rank.lean

```diff
@@ -8,6 +8,7 @@ import Mathlib.LinearAlgebra.Matrix.ToLin
 import Mathlib.LinearAlgebra.FiniteDimensional
 import Mathlib.LinearAlgebra.Matrix.DotProduct
 import Mathlib.Data.Complex.Module
+import Mathlib.Data.IsROrC.Basic
```

Comment:
This is too heavy an import for this file. If you need it, it means the results don't belong here.